pfcpiface/up4.go

```diff
@@ -377,30 +370,7 @@ func (up4 *UP4) setUpfInfo(u *upf, conf *Conf) {
 }
 
 func (up4 *UP4) clearAllTables() error {
-	tables := []string{TableUplinkSessions, TableDownlinkSessions, TableUplinkTerminations, TableDownlinkTerminations, TableTunnelPeers, TableApplications}
-	tableIDs := make([]uint32, len(tables))
-
-	for _, table := range tables {
-		tableID, err := up4.p4RtTranslator.getTableIDByName(table)
-		if err != nil {
-			return err
-		}
-
-		tableIDs = append(tableIDs, tableID)
-	}
-
-	err := up4.p4client.ClearTables(tableIDs)
-	if err != nil {
-		return err
-	}
-
-	interfacesTableID, err := up4.p4RtTranslator.getTableIDByName(TableInterfaces)
-	if err != nil {
-		return err
-	}
-
-	err = up4.p4client.ClearTable(interfacesTableID)
-	if err != nil {
+	if err := up4.p4client.ClearTables(p4constants.GetTableIDList()); err != nil {
```

Comment:
this is not safe IMO - we might not always want to clear ALL tables.

Reply:
it's easy to forget about updating this part when we will add new P4 tables

Reply:
Well, then the method should not be called `ClearAllTables()` but something like `ClearSomeTables()` 😄 . Maybe `ClearTablesBatch()` ?

Reply:
We can just call this `ClearTables()` then

Reply:
Done